fenix/app/src/main/res/values/strings.xml

```diff
@@ -1312,7 +1312,9 @@
     <!-- Title text of a delete browsing data dialog. -->
     <string name="delete_history_prompt_title">Time range to delete</string>
     <!-- Body text of a delete browsing data dialog. -->
-    <string name="delete_history_prompt_body">Removes history (including history synced from other devices), cookies and other browsing data.</string>
+    <string name="delete_history_prompt_body" moz:RemovedIn="130" tools:ignore="UnusedResources">Removes history (including history synced from other devices), cookies and other browsing data.</string>
```

Comment:
@mavduevskiy @rocketsroger This should be the current Nightly version which is 115 https://whattrainisitnow.com/.

Reply:
@gabrielluong
Should it? And then we would add it again? I mentioned in the bug that we plan to roll back to the previous string once the follow ups are finished.

Reply:
It should be fine based on what Roger mentioned. It's mostly weird to see what we have a string for removal for something way in the future with the added nuance that we might keep the string if some additional changes happen.

Reply:
@gabrielluong 
What would be the ideal approach in your opinion? Going without a removed tag?

Reply:
Right, I think it would be easier to not have the mozRemovedIn label if we know that we want to keep it at the end of the day. However, my understanding of the issue is that *if* we fix the bug then we will need the string again - makes it tougher call so it's dependent on your confidence on whether or not we will fix the bug.